doc/usage.md

```diff
@@ -315,6 +315,8 @@ inputs:
 19. --result_filetag: name of tag appended to the result filename. Default is 'fritzUpload'.
 20. --result_format : result file format; one of csv, h5 or parquet. Default is parquet.
 21. --replace_classifications : flag to delete each source's existing classifications before posting new ones.
+22. --radius_arcsec: photometry search radius for uploaded sources.
+23. --no_ml: flag to post classifications that do not originate from an ML classifier.
```

Comment:
Is there a context where we are posting non-ML classifications through this pathway?

Reply:
@mcoughlin I'd consider the initial training set classifications I posted to Fritz as distinct enough from subsequent ML classifications to warrant this flag.

Reply:
Ah got ya